docs/spec/spec.md

```diff
@@ -116,13 +116,16 @@ The conforming implementation of the specification is released and included in t
         * 10.1.3 [Context Initializer Function](#1013-context-initializer-function)
         * 10.1.4 [CORS Configurations](#1014-cors-configurations)
         * 10.1.5 [GraphiQL Configurations](#1015-graphiql-configurations)
-        * 10.1.6 [Service Level Interceptors](#1016-service-level-interceptors)
+        * 10.1.6 [Service Interceptors](#1016-service-interceptors)
```

Comment:
Let's check the BBEs to verify that these changes does not break the BBE links

Reply:
It seems not to break the BBE links.